### ISSUE TITLE
CalledProcessError with django-wkhtmltopdf

### ISSUE THREAD

**Issue by akashpatni** [incuna/django-wkhtmltopdf#122]
Hi ,

I am getting the below mention error:

Command '['/usr/local/bin/wkhtmltopdf', '--encoding', 'utf8', '--quiet', 'False', '/tmp/wkhtmltopdf793ytosn.html', '-']' returned non-zero exit status -6

and here is my code

from django.views.generic.base import View
from wkhtmltopdf.views import PDFTemplateResponse

class MyPDFView(View):
    template=settings.BASE_DIR+'/templates/frontend/pdf.html'
    context= {'title': 'Hello World!'}

```
def get(self, request):
    response = PDFTemplateResponse(request=request,
                                   template=self.template,
                                   filename="hello.pdf",
                                   context= self.context,
                                   show_content_in_browser=False,
                                   )
    return response
```

and in settings.py

INSTALLED_APPS = (
    'wkhtmltopdf',
)

WKHTMLTOPDF_CMD = '/usr/local/bin/wkhtmltopdf'

WKHTMLTOPDF_CMD_OPTIONS = {
    'quiet': False,
}

Any help on this.

Thanks

**Comment by chidg** [comment]
Hi,
try removing `'quiet': False` from your options. The quiet option is false by default, so you only need to specify the option if you are making it true, and the mapping of this options dictionary to the command line options results in `--quiet False` being passed in the command, which is incorrect syntax.

**Comment by tclancy** [comment]
>mapping of this options dictionary to the command line options results in --quiet False being passed in the command, which is incorrect syntax.

I don't understand this. If I specify `'quiet': True` it works fine, but when I swap it to `False` I get an error. What's the point of exposing the option if it causes an error every time? (I'm assuming setting to `False` will give me any warnings or info about things I may not have noticed and I'd like to see those.)

**Comment by johnraz** [comment]
I'm gonna assume that this was fixed by #134, if not please re-open.

**Comment by tclancy** [comment]
Yeah, this is set, thanks.
